stock/templates/registered/hrapplicationview.html

```diff
@@ -50,6 +50,13 @@ <h1 class="page-header text-center">View Application</h1>
                             </div>
                         </div>
                         <div class="form-group">
+                            <label class="control-label" for="id_primary_email">Primary Email Address</label>
+
+                            <div class=" ">
+                                <input class=" form-control" value="{{ application.primary_email }}" disabled>
```

Comment:
You can get the applicant's email by referencing `application.user.email` - no model or form changes needed.